pkg/autoscaler/autoscaler.go

```diff
@@ -261,29 +258,28 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (int32, bool) {
 			if lastStat[stat.PodName].Time.Before(*stat.Time) {
 				lastStat[stat.PodName] = stat
 			}
-			// Update lastRequestTime if the current stat is newer and
-			// actually contains requests
-			if a.lastRequestTime.Before(*stat.Time) && stat.RequestCount > 0 {
-				a.lastRequestTime = *stat.Time
+
+			if a.firstMetricObserved == nil || a.firstMetricObserved.After(instant) {
+				a.firstMetricObserved = &instant
 			}
 		} else {
 			// Drop metrics after 60 seconds
 			delete(a.stats, key)
 		}
 	}
 
+	// Scale to 1 by default.
+	if a.firstMetricObserved == nil {
+		logger.Debug("No metrics have arrived yet.")
+		return 1, false
```

Comment:
The `false` here indicates that the autoscaler cannot recommend a scale.  Can you change this to `return 0, false` just like below?